### DISCUSSION TITLE
Unraid Wiki to Install Scrypted is false

### DISCUSSION THREAD

**Discussion by NeZios** [koush/scrypted#1198] [Q&A]
**Describe the bug**
I wanted to install Scrypted on my Unraid Server, using the template of the Docker App Store and the network type "br0" as suggested on the Wiki, make getting access to Scrypted impossible through the webUI

**To Fix the issue**

Using the Network type : **HOST** make Scrypted accessible perfectly. 

**Screenshots**
See directly from the official wiki for Unraid :
https://github.com/koush/scrypted/wiki/Installation:-Docker-Unraid

**Comment by ryphez** [comment]
This method doesn't allow custom docker networks. The issue is the port isn't explicitly passed in the template. I'm not sure what 10443 maps to internally but mapping 10443 external to some internal port would help alleviate this
